knexfile.js

```diff
@@ -0,0 +1,48 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'postgresql',
+    connection: {
+      database:  'react_kanban',
+      host:      'localhost',
+      user:      'zbuzzeo',
+      password:  'password',
+    }
+  },
+
+  staging: {
+    client: 'postgresql',
+    connection: {
+      database: 'my_db',
+      user:     'username',
+      password: 'password'
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  },
+
+  production: {
+    client: 'postgresql',
```

Comment:
I believe Jason went over how to update this to use .env rather than hardcoded strings.